src/util/pg.py

```diff
@@ -610,6 +614,15 @@ def remove_column(cr, table, column, cascade=False):
 
 
 def alter_column_type(cr, table, column, type, using=None, where=None, logger=_logger):
+    if where and not using:
```

Comment:
I think we should add docs to this function to clarify the usage of each parameter. I feel we need something like "`where` is used to restrict the values converted via `using`"...

Reply:
The drop of views should be documented if we add the docs.